pkg/server/util.go

```diff
@@ -124,28 +125,41 @@ func resizeVolume() error {
 	// if we have a partitioned device then we need to grow the root partition
 	// otherwise the disk has no partitions and we can skip this step
 	if rootDevice != rootPartition {
-		partitionNumber, err := getPartitionNumber(rootPartition)
-		if err != nil {
-			return err
-		}
-		out, err = exec.Command("growpart", rootDevice, partitionNumber).CombinedOutput()
-		if err != nil {
+		var result error
+		for count := 0; count < 30; count++ {
+			partitionNumber, err := getPartitionNumber(rootPartition)
```

Comment:
Gotta say something so I'll say this: this could get moved outside the for loop.  Doesn't have to but it could. That's all I got.

Reply:
> Gotta say something so I'll say this: this could get moved outside the for loop. Doesn't have to but it could. That's all I got.

Totally, it's dumb to keep it inside the loop :smiley_cat: